Description: Use SIGTERM instead of SIGHUP for graceful shutdown
 The SIGHUP couchdb uses by default for a graceful shutdown does not
 work reliably in Debian as apt, when installing the package, SigIgn
 masks SIGHUP for all its child processes. This is intentional and hard
 to work around, and CouchDB does a graceful shutdown with SIGTERM as
 well so there is no reason to use SIGHUP.
Author: Dominik George <nik@naturalnet.de>
Bug-Debian: http://bugs.debian.org/692295

---

--- couchdb-1.2.0.orig/bin/couchdb.tpl.in
+++ couchdb-1.2.0/bin/couchdb.tpl.in
@@ -272,7 +272,7 @@ stop_couchdb () {
             echo > $PID_FILE
         fi
         if kill -0 $PID 2> /dev/null; then
-            if kill -1 $PID 2> /dev/null; then
+            if kill -15 $PID 2> /dev/null; then
                 if test "$1" = "false"; then
                     echo "Apache CouchDB has been shutdown."
                 else
